docs/UserGuide.md

```diff
@@ -666,7 +644,19 @@ Here are potential error messages that you may receive and here's how to fix the
 
 | Error Message                     | Why it happens                                   | Fix                                                                                                                                                                                                                   |
 |-----------------------------------|--------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Leave records could not be saved! | You do not have the permission to write the file | Try renaming your file name when typing out the command, especially if the previous name refers to an existing file. If not, move HRMate to a different folder where you can create files, and run the command again. |      
+| Leave records could not be saved! | You do not have the permission to write the file | Try renaming your file name when typing out the command, especially if the previous name refers to an existing file. If not, move HRMate to a different folder where you can create files, and run the command again. |
+--------------------------------------------------------------------------------------------------------------------
+
+### Exit HRMate
+Once you've finished your work, you might want to close HRMate. If you love typing over
+clicking, you can close the app via keyboard too!
+
+#### Close the application: `exit`
+Here's how you can close HRMate using your keyboard:
+
+1. In HRMate, type this command into the [command box](#command-box): `exit` and press enter.
+2. That's it! HRMate will now close, and you will find your employee records and leave records there the next time you
+reopen HRMate.
```

Comment:
Same with the above comment on the clear command

Reply:
Change heading size for consistency